try:
    a=5/0
except:
    print('arithmetic exception')
